profiles/search.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import os
+import json
+
+class Search:
+    
+    def __init__(self):
+        self.currentEmployeePage = 1
+        self.currentUserPage = 1
+    
+    # LinkedIn Search is restricted to certain amount of people,
+    # Can use google to search instead.
+    def get_linkedin_urls_google_search(self, company, title):
```

Comment:
Like above mentioned, add another parameter in here that specifies which page to look at.